chia/wallet/puzzles/p2_delegated_secp.clsp

```diff
@@ -0,0 +1,14 @@
+; p2_delegated with SECP256-R1 signature
+; this is the "standard puzzle" for spending coins with SECP keys (ie secure enclave)
+
+(mod (SECP_PK delegated_puzzle delegated_solution signature)
+  (include *standard-cl-21*)
+  (include condition_codes.clib)
+  (include sha256tree.clib)
+
+  (if (secp256r1_verify SECP_PK (sha256 (sha256tree delegated_puzzle)) signature)
+      ; secp256r1_verify returns 0 if successful.
+      (x)  ; this doesn't actually run because secp256_verify will raise on failure
+      (a delegated_puzzle delegated_solution)
+  )
```

Comment:
We'd have to do `assert not` since (somewhat confusingly) the verify returns 0 for success and raises for failure. But I think having `assert not secp_verify` diminishes readability